### ISSUE TITLE
[Discuss] Display total number of digital objects from a collection in DCS

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1663]
**Story**

Currently the Extent of Digitization field under Description on an object page, simply states if something is "Completely digitized" or "Partially Digitized". We need to enhance this statement to include the number of child images associated with a given parent. 

Examples
- “This collection is completely digitized with 351 digital items”
- “This collection is partially digitized with 50 digital items”
- If unknown, simply display “This collection is partially digitized”

**Acceptance**
- [ ] Statements that include number of child images attached to parent object, appear in the Extent of Digitization field under Description

**Comment by motropuk** [comment]
Waiting on additional information from Archival Context Team later today

**Comment by motropuk** [comment]
@jpengst We need to put this ticket on hold for now as the Archival Context Team need to rethink what they want here. We understood the requirement here was to display number of child images for an object. However, they want this to display how many other digitized objects from the same collection are also digitized, which would need to be pulled from ASpace I think.

**Comment by jpengst** [comment]
Ah, ok! Thanks for the update. I'll put this back in Backlog. Thanks!

**Comment by motropuk** [comment]
@alisonclemens will start joining sprints again starting next sprint, so we can work on this next sprint

**Comment by motropuk** [comment]
Moving this to in progress as a meeting about this was had on Monday and there is another today
